gemfiles/rack_1.5.2.gemfile

```diff
@@ -1,33 +1,33 @@
 # This file was generated by Appraisal
 
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'rack', '1.5.2'
+gem "rack", "1.5.2"
```

Comment:
You're right, these need to be added to the rubocop.yml like so:

<img width="1006" alt="screen shot 2016-07-26 at 11 26 05 am" src="https://cloud.githubusercontent.com/assets/542335/17144151/d0c706d0-5323-11e6-86fa-04d12db1bdf4.png">